lib/scrum/sprint_cleaner.rb

```diff
@@ -2,17 +2,17 @@ module Scrum
   class SprintCleaner < TrelloService
     include ScrumBoards
 
-    def cleanup(board_id, target_board_id)
+    def cleanup(board_id, target_board_id, labelled)
```

Comment:
Please use a named parameter instead of a boolean. When you read a call such as `cleanup(some_id, other_id, false)` you don't know what the `false` is doing.